tally_ho/apps/tally/tests/views/data/test_tally_list_view.py

```diff
@@ -0,0 +1,99 @@
+import json
+from django.test import RequestFactory
+
+from tally_ho.apps.tally.views.data import tally_list_view as views
+from tally_ho.libs.permissions import groups
+from tally_ho.libs.tests.test_base import create_tally, TestBase
+
+
+class TestTallyListView(TestBase):
```

Comment:
It would be nice to have docstrings in this module :)

Reply:
haha is it necessary to add doc strings to test classes?

Reply:
:) arguably yes

Reply:
hmm i don't think so, because even the django project test classes do not have doc strings. I think it's because the classes being tested have doc strings and also the test cases names are clear enough to describe whats being tested. check this https://github.com/django/django/blob/89032876f427a77ab4de26493190280377567d1c/tests/admin_registration/tests.py#L19

Reply:
Well yes.  I would still say "arguably yes" - and it is entirely at your discretion.

I'm for the idea because:

- tests to me are also some kind of documentation and if the bare code isn't clear, I would try check comments, docstrings if they existed.
- if you run pylint here it would fail

Reply:
cool that's a good point, this makes sense to me now :)

Reply:
Done